azure-pipelines.yml

```diff
@@ -0,0 +1,50 @@
+# Python package
+# Create and test a Python package on multiple Python versions.
+# Add steps that analyze code, save the dist with the build record, publish to a PyPI-compatible index, and more:
+# https://docs.microsoft.com/azure/devops/pipelines/languages/python
+trigger: none
+
+pool:
+  vmImage: 'ubuntu-latest'
+strategy:
+  matrix:
+    Python37:
+      python.version: '3.7'
+    Python38:
+      python.version: '3.8'
+
+steps:
+- task: UsePythonVersion@0
+  inputs:
+    versionSpec: '$(python.version)'
+  displayName: 'Use Python $(python.version)'
+
+- script: |
+    python -m pip install --upgrade pip
+    pip install -r requirements.txt
+    pip install -r requirements_dev.txt
+    pip install twine
+  displayName: 'Install dependencies'
+
+- script: |
+    pip install pytest pytest-azurepipelines==1.0.0rc5
+    pytest --cov=omigami tests/ --cov-report html --nunitxml test-output.xml
+  displayName: 'Run tests'
+
+- script: |
+    make docs
+  displayName: 'Build documentation'
+
+# TODO: these next two are to push to RTD and pypi but the logic is a WIP
```

Comment:
the actual push will go in the two script section right?

Reply:
exactly